src/tools/lambert.rs

```diff
@@ -40,10 +43,87 @@ pub struct LambertSolution {
     pub phi: f64,
 }
 
-/// Solves the Lambert boundary problem using a standard secant method.
+/// Calculate the direction multiplier based on the transfer kind.
+///
+/// # Arguments
+///
+/// * `kind` - The kind of transfer (auto, short way, long way, or number of revolutions).
+/// * `r_final` - The final radius vector.
+/// * `r_init` - The initial radius vector.
+///
+/// # Returns
+///
+/// `Result<f64, NyxError>` - The direction multiplier or an error if the transfer kind is not supported.
+fn calculate_dm(
```

Comment:
I think this could be in the `impl` of `TransferKind`, something like this:

```rust
impl TransferKind {
    fn direction_of_motion(self, r_final: &Vector3<f64>, r_init: &Vector3<f64>) -> Result<f64, NyxError> {
        // Do the same math
    }
}
```

Then you would call it directly like so: `kind.direction_of_motion(&r_final, &r_init)`.

Reply:
Done.